lib/pysquared/logger.py

```diff
@@ -24,35 +52,39 @@ def _log(self, level: str, message: str, **kwargs) -> None:
 
         json_output = json.dumps(kwargs)
 
-        if self.logToStandardOut:
+        if self.log_to_standard_out and self.can_print_this_level(level_value):
             print(json_output)
 
     def debug(self, message: str, **kwargs) -> None:
         """
         Log a message with severity level DEBUG.
         """
-        self._log("DEBUG", message, **kwargs)
+        self._log("DEBUG", 1, message, **kwargs)
 
     def info(self, message: str, **kwargs) -> None:
         """
         Log a message with severity level INFO.
         """
-        self._log("INFO", message, **kwargs)
+        self._log("INFO", 2, message, **kwargs)
 
     def warning(self, message: str, **kwargs) -> None:
         """
         Log a message with severity level WARNING.
         """
-        self._log("WARNING", message, **kwargs)
+        self._log("WARNING", 3, message, **kwargs)
 
     def error(self, message: str, **kwargs) -> None:
         """
         Log a message with severity level ERROR.
         """
-        self._log("ERROR", message, **kwargs)
+        self.error_counter.increment()
+        self._log("ERROR", 4, message, **kwargs)
 
     def critical(self, message: str, **kwargs) -> None:
         """
         Log a message with severity level CRITICAL.
         """
-        self._log("CRITICAL", message, **kwargs)
+        self._log("CRITICAL", 5, message, **kwargs)
+
+    def get_error_count(self) -> int:
+        return self.error_counter.get()
```

Comment:
It looks like this isn't used anywhere but if we make the class member vars private then we will need this getter.